localization/english/culture/wc_culture_eras_l_english.yml

```diff
@@ -0,0 +1,5 @@
+﻿l_english:
+
+ #ERAS
+ culture_era_early_modern:0 "Early Modern"
+ culture_era_early_modern_desc:0 "The Early Modern Era marks the beginning of a new era. The old ways have been forgotten as a new age of enlightenment dawns on the people of Azeroth."
```

Comment:
"The Era marks an era." That's a tautology.
Also, "new era" and in the next sentence "new age".